_pages/install/cors.md

```diff
@@ -23,6 +23,8 @@ You can use various CORS headers to specify whether the origin is allowed, with
 
 In development, the allowed origins are often configured with an asterisk (`*`), which allowlists all clients, regardless of their domain. In a production environment, this should contain the different domains that are allowed to interact with the back end API.
 
+Since SAP Commerce Cloud version 2105, it is not allowed to use the combination of the allowCredentials=true and allowedOrigins=* attributes. In these cases, use allowedOriginPatterns=* instead.
+
```

Comment:
Would it be useful/relevant to add a link to the commerce cloud doc about this? Like https://help.sap.com/docs/SAP_COMMERCE/d0224eca81e249cb821f2cdf45a82ace/b27d995150a74be08869e60e3fbc7395.html?version=2105&locale=en-US

Reply:
Agreed, link added 🙂 
Thank you again for this review! 🙇